GPU/GLES/StateMappingGLES.cpp

```diff
@@ -229,15 +229,7 @@ void DrawEngineGLES::ApplyDrawState(int prim) {
 
 #ifndef USING_GLES2
 			if (gstate_c.Supports(GPU_SUPPORTS_LOGIC_OP)) {
-				// TODO: Make this dynamic
-				// Logic Ops
-				if (gstate.isLogicOpEnabled() && gstate.getLogicOp() != GE_LOGIC_COPY) {
-					// TODO: Fix logic ops.
-					//glstate.colorLogicOp.enable();
-					//glstate.logicOp.set(logicOps[gstate.getLogicOp()]);
-				} else {
-					//glstate.colorLogicOp.disable();
-				}
+				renderManager->SetLogicOp(gstate.isLogicOpEnabled(), logicOps[gstate.getLogicOp()]);
```

Comment:
Hmm, I don't have the game from #3389, but it seems like GE_LOGIC_COPY may have been intentionally treated as disabled?

-[Unknown]

Reply:
I think that was just meant as an optimization that probably isn't really relevant as hardware that can do logic ops is probably pretty fast at it... but who knows.

Reply:
I thought it was an issue with logic ops and alpha blending being enabled at the same time, which I believe works on the PSP.

-[Unknown]

Reply:
Ah, could be yes. I'll reintroduce the check.